test/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This should not be here. I think there is a new repository defined under the test folder

Reply:
Sorry, it was my mistake. It has been corrected.